Dispatcher/UI/Profile/Settings/SettingsViewModel.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+class SettingsViewModel {
+    
+    let repository = SettingsRepository()
+    
+    var appSettings: [SettingModel] = [
+        SettingModel(sectionTitle: "Search results", options: [
+            SingleSetting(title: "Save filters", description: "Allow us to save filters when entering back to the app", status: .off),
+            SingleSetting(title: "Save search results", description: "Allow us to save your search result preferences for next search", status: .off)
+        ]),
+        SettingModel(sectionTitle: "App preferences", options: [
+            SingleSetting(title: "Notification", status: .on)
+        ]),
+    ]
+    
+    func fetchUserSettingsPreferences() {
+        appSettings[0].options[0].status = repository.fetchSaveFiltersSetting()
```

Comment:
fetching is usually refers to get remote data (from server..).
I would change the name to 'getSaveFiltersStatus'.

same with the others below.

Reply:
Fixed in all places
@abenartz